src/imitation/scripts/common/demonstrations.py

```diff
@@ -90,6 +91,10 @@ def load_expert_trajs(
     Raises:
         ValueError: There are fewer trajectories than `n_expert_demos`.
     """
+    # wrap with str() in case rollout_path is a Path object
+    if "huggingface" in str(rollout_path):
```

Comment:
Would it not be better to check more explicitly where within the path it is? e.g. you could do `rollout_path.name`.

Reply:
Do we want to overload `rollout_path` in this way? An alternative would be to add an explicit `rollout_type` that can be e.g. `disk` or `huggingface`, similar to what we did for `policy_type`.